core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -224,27 +217,16 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
       TestUtils.waitAndVerifyAcls(TopicReadAcl ++ TopicWriteAcl ++ TopicDescribeAcl, s.apis.authorizer.get, topicResource)
       TestUtils.waitAndVerifyAcls(GroupReadAcl, s.apis.authorizer.get, groupResource)
     }
-    //Produce records
-    debug("Starting to send records")
     sendRecords(numRecords, tp)
-    //Consume records
-    debug("Finished sending and starting to consume records")
   }
 
   /**
     * Tests that a producer fails to publish messages when the appropriate ACL
     * isn't set.
     */
-  @Test
+  @Test(expected = classOf[TimeoutException])
   def testNoProduceWithoutDescribeAcl {
-    //Produce records
-    debug("Starting to send records")
```

Comment:
But I guess you can add them back when you need them easily enough? I found them a little distracting when reviewing 3396 and some of them were even incorrect or misleading (which made me think they had probably just been copied from an existing test).